### PR TITLE
Adding flavor and languages for ICM-PH

### PR DESCRIPTION
**Pull request by binokaryg**
Adding icm_ph_chc flavor medic/config-icm#84

Adding strings for prominent location access disclosure in:
- Tagalog [tel]
- Illonggo [hil]
- Bisaya [ceb]
- 
Modifying strings for the disclosure in:
- Nepal [ne]

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -32,4 +32,4 @@ jobs:
         ANDROID_KEY_PASSWORD: ${{ secrets.ANDROID_KEY_PASSWORD }}
       with:
         lane: build
-        options: '{ "flavor": "unbranded" }'
+        options: '{ "flavor": "icm_ph_chc" }'
```
Comment (mrsarm) [inline .github/workflows/build.yml:35]: There is no need to replace the default flavor here.

File: Makefile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ADB = ${ANDROID_HOME}/platform-tools/adb
 GRADLEW = ./gradlew
-flavour = UnbrandedWebview
+flavour = icm_ph_chc
```
Comment (mrsarm) [inline Makefile:3]: No need to replace the default flavor to build another.

**Review by mrsarm — CHANGES_REQUESTED** [review]

File: .github/workflows/build.yml (hunk 1 above)
Comment (binokaryg) [reply .github/workflows/build.yml]: Thanks, I have reverted the change.

File: Makefile (hunk 2 above)
Comment (binokaryg) [reply Makefile]: Thanks, I have reverted the change.

File: src/main/res/values-ne/strings.xml [hunk 3]
```diff
@@ -3,6 +3,6 @@
 	<string name="locRequestTitle">स्थान पहुँच</string>
 	<string name="locRequestMessage">यो अनुप्रयोगले विश्लेषण गर्दछ र तपाईंको क्षेत्रमा स्वास्थ्य परिणामहरू सुधार गर्न स्थान डाटा संकलन गर्दछ। स्थान पहुँचलाई अनुमति दिन "खोल्नुहोस्" चयन गर्नुहोस्, र त्यसपछि अर्को प्रॉम्प्ट पुष्टि गर्नुहोस्।</string>
 	<string name="locRequestOkButton">खोल्नुहोस्</string>
-	<string name="locRequestDenyButton">धन्यबाद</string>
+	<string name="locRequestDenyButton">हुँदैन, धन्यबाद</string>
```
Comment (mrsarm) [inline src/main/res/values-ne/strings.xml:6]: Thanks for this correction

**Review by mrsarm — APPROVED** [review]
LGTM

**Comment by mrsarm** [comment]
@binokaryg the PR is approved but we couldn't pass it through QA yet so I going to pause the merge until we release the new version with this other PR that is urgent: https://github.com/medic/medic-android/issues/148

**Comment by mrsarm** [comment]
> @binokaryg the PR is approved but we couldn't pass it through QA yet so I going to pause the merge until we release the new version with this other PR that is urgent: #148

Forget about this, I talked with @ngaruko and we are going to include it in the release, merging ...

**Pull request merged**